war/src/main/webapp/css/responsive-grid.css

```diff
@@ -8,24 +8,6 @@
  * Only the grid and responsive rules from bootstrap.
  */
 
-* {
-  -webkit-box-sizing: border-box;
```

Comment:
Is it fine that the browsers specifics have been removed?

Reply:
Looks like it https://github.com/jenkinsci/jenkins/pull/4697#discussion_r417586199

Reply:
https://github.com/jenkinsci/jenkins/pull/4697#discussion_r417586199